Package.swift

```diff
@@ -4,12 +4,12 @@ import PackageDescription
 let package = Package(
     name: "Haptico",
     platforms: [
-        .iOS(.v9),
+        .iOS(.v13),
```

Comment:
Won't it make it impossible to use Haptico on iOS prior to iOS 13?

Reply:
Glad you asked. I had changed that to require iOS 13 then went back and fixed it where it could still be compatible with 9+ and forgot to change the required version back. I pushed a new commit to update that back to v9. The changes I made to support the newer devices would require iOS 13+ but those devices were shipped with OS higher than 13 and I did wrap the changes in if #available(iOS 13.0, *) so it should be fine.